src/index.html

```diff
@@ -10,23 +10,36 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Airbnb Style Search Bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <section
+      class="input input--large"
+      data-qa="big"
+    >
+      <input
+        class="input__field input__field--large"
+        type="text"
+        name="big-search"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
```

Comment:
According to the checklist, there should be an empty line between multiline sibling blocks of HTML. Please add an empty line between the two `section` elements for better readability.